src/modules/simulations/actions.js

```diff
@@ -7,17 +7,10 @@ import {GraphPropagation} from '../../lib/propagation/graph-propagation'
 
 const delay = (ms) => new Promise(resolve => setTimeout(resolve, ms))
 
-export function* runMetricSimulation({getState, metricId, dispatch, runAllUnsimulated}) {
-  if (runAllUnsimulated) {
-    const metric = e.metric.get(getState().metrics, metricId)
-    const spaceId = !!metric && metric.space
-    const propagation = new GraphPropagation(dispatch, getState, {spaceId, onlyUnsimulated: true})
-    yield propagation.run()
-  } else {
-    const propagation = new GraphPropagation(dispatch, getState, {metricId, onlyHead: true})
-    yield propagation.run()
-    yield* runFormSimulation({getState, metricId, dispatch})
-  }
+export function* runFormSimulation({getState, metricId, dispatch}) {
+  const propagation = new GraphPropagation(dispatch, getState, {metricId, onlyHead: true})

```

Comment:
We do this in canvas editing so that resimulating the card being edited is quick, but the rest of the chain still updates later. I'd rather not pull out those two lines into a separate function at this point, as I don't ever anticipate a need for that functionality to be used separately.